lang/java/tools/pom.xml

```diff
@@ -145,7 +145,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-core</artifactId>
```

Comment:
Are you sure that hadoop-core is not mandatory for a specific case ? 
if it is used in some cases ; we have just to add hadoop-client as dependency (not replace hadoop-core).